doc/user/overview.rst

```diff
@@ -121,17 +100,23 @@ An illustration of the large scale architecture is given below.
    +-------------+   +------------------+   +-------------+
 
 
-The multi-process architecture brings extensibility, modularity and
-maintainability.  All of the FRR daemons can be managed through a single
-integrated user interface shell called *vtysh*.  *vtysh* connects to each
-daemon through a UNIX domain socket and then works as a proxy for user input.
-In addition to a unified frontend, *vtysh* also provides the ability to
-configure all the daemons using a single configuration file through the
-integrated configuration mode avoiding the problem of having to maintain a
-separate configuration file for each daemon.
+All of the FRR daemons can be managed through a single integrated user
+interface shell called *vtysh*. *vtysh* connects to each daemon through a UNIX
+domain socket and then works as a proxy for user input. In addition to a
+unified frontend, *vtysh* also provides the ability to configure all the
+daemons using a single configuration file through the integrated configuration
+mode. This avoids the overhead of maintaining a separate configuration file for
+each daemon.
+
+FRR is currently under heavy development to implement a new internal
```

Comment:
nit: "under heavy development" - maybe "working actively to implement" or "currently implementing" ?

Reply:
changed